content/docs/tasks/security/authn-policy/index.md

```diff
@@ -570,6 +570,96 @@ $ for i in `seq 1 10`; do curl --header "Authorization: Bearer $TOKEN" $INGRESS_
 401
 {{< /text >}}
 
+### End-user authentication with per-path requirements
+
+End-user authentication can be enabled or disabled based on request path. This is useful if you want to
+whitelist some paths from End-user authentication, like for paths used for health check or status report.
+You can also specify different JWT requirements on different request paths.
+
+#### Whitelist paths from End-user authentication
```

Comment:
Whitelist ... => Disable End-user authentication for specific paths

Applies to other places as well. Do not call it "whitelist". It is not very clear what it means.